product-tours/tour-pytorch/Tour-with-PyTorch.ipynb

```diff
@@ -0,0 +1,593 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "exp = neptune.get_experiment()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "AW4MM4qj5ErX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "#### Stop Neptune experiment after training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Below method is necessary only for notebooks users. In the Python scipts experiment is closed automatically when script finishes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "# check logs\n",
+    "correct_logs_set = {'batch_loss', 'batch_acc', 'predictions'}\n",
+    "from_exp_logs = set(exp.get_logs().keys())\n",
+    "\n",
+    "assert correct_logs_set == from_exp_logs, '{} - incorrect logs'.format(exp)\n",
+    "\n",
+    "# check parameters\n",
+    "assert set(exp.get_parameters().keys()) == set(PARAMS.keys()), '{} parameters do not match'.format(exp)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 71
+    },
+    "colab_type": "code",
+    "id": "GvDSBSrOx-R4",
+    "outputId": "667d1b2f-cdef-44f0-f505-832205dea223",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Summary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Now, go back to the previously opened browser tab with your experiment to see:\n",
+    "\n",
+    "* tracked [metrics](https://ui.neptune.ai/o/shared/org/tour-with-pytorch/e/TOURTORCH-2/charts) (loss, accuracy),\n",
+    "* [image predictions](https://ui.neptune.ai/o/shared/org/tour-with-pytorch/e/TOURTORCH-2/logs),\n",
+    "* [parameters](https://ui.neptune.ai/o/shared/org/tour-with-pytorch/e/TOURTORCH-2/parameters)\n",
+    "* saved [model weights](https://ui.neptune.ai/o/shared/org/tour-with-pytorch/e/TOURTORCH-2/artifacts).\n",
+    "\n",
+    "You just learned how to:\n",
+    "* set project and create experiment using Neptune API,\n",
+    "* log PyTorch meta-data the experiment.\n",
+    "\n",
+    "Such logging is a basic usage of Neptune to track PyTorch experiments."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## If you want to learn more, go to the [Neptune documentation](https://docs.neptune.ai/integrations/pytorch.html)."
```

Comment:
I don't think this link works (as it is in the header)

Reply:
changed to text

Reply:
But wait, I just checked it in Jupyter environment and it works.
Similar when I opened notebook file in the github. I think we can keep it like this.

Did yo uhave problems with the link?